.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "servant"]
+	path = servant
```

Comment:
also, we now still need to call `cabal sandbox add-source` for this, right?  is there a way around that?  if not: it needs to be added to the docs.